storage/replica.go

```diff
@@ -594,10 +592,10 @@ func (r *Replica) addAdminCmd(ctx context.Context, args proto.Request) (proto.Re
 
 	switch tArgs := args.(type) {
 	case *proto.AdminSplitRequest:
-		resp, err := r.AdminSplit(*tArgs)
+		resp, err := r.AdminSplit(*tArgs, r.Desc())
```

Comment:
I actually mentioned this in the commit message; this pathway should only be used if an admin is manually splitting a range, internal pathways (e.g. "maybeSplitOnZoneConfig" or whatever) should just call AdminSplit directly.  

However, i'm saving that change for a future commit - i'll create an issue for it.